store/sqlstore/post_store.go

```diff
@@ -547,7 +547,7 @@ func (s *SqlPostStore) GetPosts(channelId string, offset int, limit int, allowFr
 	})
 }
 
-func (s *SqlPostStore) GetPostsSince(channelId string, time int64, allowFromCache bool) store.StoreChannel {
+func (s *SqlPostStore) GetPostsSince(channelId string, time int64, limit int, allowFromCache bool) store.StoreChannel {
```

Comment:
Also just a question about this although it doesn't matter much since it's an edge case, but how does this limit work? Does it get the 1000 oldest posts since the given time or the 1000 most recent posts since the given time?

Reply:
It gets the 1000 (or the limit) most recent posts since the given time.